src/pudl_archiver/zenodo/entities.py

```diff
@@ -81,6 +82,17 @@ def check_empty_string(cls, doi: str):  # noqa: N805
     def from_data_source(cls, data_source_id: str) -> "DepositionMetadata":
         """Construct deposition metadata object from PUDL DataSource model."""
         data_source = DataSource.from_id(data_source_id)
+        creators = [
```

Comment:
Some new data sources don't have contributors defined in PUDL

Reply:
Makes sense, and then we can just claim credit :)